tests/ui/suggestions/args-instead-of-tuple-errors.stderr

```diff
@@ -64,6 +71,13 @@ LL |     let _: Option<(i32,)> = Some(5_usize);
    |
    = note: expected tuple `(i32,)`
                found type `usize`
+help: the return type of this call is `usize` due to the type of the argument passed
+  --> $DIR/args-instead-of-tuple-errors.rs:14:29
+   |
+LL |     let _: Option<(i32,)> = Some(5_usize);
+   |                             ^^^^^-------^
+   |                                  |
+   |                                  this argument influences the return type of `Some`
```

Comment:
Maybe what we should do is tweak the *wording* depending on the item type? Because I'm sure people are going to be confused by talking about the "return type of an enum variant". Seeing tuple structs as functions is natural from the compiler's point of view, but not from the users'.

Reply:
I've put something in that tweaks the wording. It's slightly awkward for constructors, because they return a type that's different from the argument, and I don't know if there is a reliable way of getting that type regardless of wrapping, but this now is at least coherent.